src/services/utils.js

```diff
@@ -2,6 +2,16 @@ var dom = require('@nymag/dom'),
   _ = require('lodash'),
   references = require('./references');
 
+/**
+ * Checks that the string is a route while accounting for underscored routes
+ * @param {string} string
+ * @param {string} route
+ * @return {boolean}
+ */
+function checkUri(string, route) {
+  return string.indexOf('/' + route + '/') > -1 ||  string.indexOf('/_' + route + '/') > -1;
```

Comment:
could use regex to simplify slightly?
`return !!string.match(/new RegExp('[_]?' + route/)`

Reply:
maybe that makes it less readable though : /

Reply:
I like regex. If it's hard to read, a short explanatory comment doesn't go amiss.